.github/workflows/pr-python-format.yml

```diff
@@ -0,0 +1,39 @@
+name: "Check Python Formatting"
+on:
+  pull_request:
+    # run on .py
+    paths:
+      - '**.py'
+
+jobs:
+  python_formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch LLVM sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 2
```

Comment:
Why do we need `fetch-depth: 2` ?

Reply:
darker is just formatting the diff for the PR - so we need the history to compute the diff from base. 2 is actually not that bad it goes pretty quickly compared to 0.